samples/tests/Console.js

```diff
@@ -30,3 +54,20 @@ console.log(longstr);
 console.log("\nTesting a string too long...");
 longstr += '6';  // length 256
 console.log(longstr);
+
+console.log("\nTesting console.assert... ");
+console.assert(true);
+console.log("expected result: true");
+console.assert(false, "My lover!");
+console.log("expected result: false - My lover!");
+console.assert(consoleStr);
+console.log("expected result: [error] invalid parameters");
+console.assert(true, longstr);
+console.log("expected result: [error] string is too long");
```

Comment:
If the condition is true, the longstr is never looked at so it shouldn't fail or care that the string is too long.